infrastructure/logic-app/rota/hmi-la-rota-dtu.json

```diff
@@ -1101,7 +1107,7 @@
                                                                     "type": "SetVariable",
                                                                     "inputs": {
                                                                         "name": "sessionDuration",
-                                                                        "value": "180"
+                                                                        "value": "360"
```

Comment:
In PUB-909, it states that AM session should have duration of 180. The new rota code changed to 360 in one of the commit 3 above:
•	Is this expected changed?
•	Do we have change request (CR) for this?

Reply:
yes its due to lack of ROTA data in SNL